.github/workflows/regression_test_summariseEpoch.yml

```diff
@@ -0,0 +1,43 @@
+name: Python - summariseEpoch.py
+
+on: [push]
+
+jobs:
+  build:
```

Comment:
the name of this job shouldn't be `build` because it is trying to check the summary. Maybe something `summary-check` would be better?